### PR TITLE
fix onBackgroundClick event handling when enablePanZoom=false

### PR DESCRIPTION
**Pull request by guillaume**
`onBackgroundClick` event is never reached if `enablePanZoom=false` 

This bug can be noticed in the continents example where clicking in the background won't remove the selection http://software.es.net/react-timeseries-charts/#/example/continents

https://github.com/esnet/react-timeseries-charts/blob/master/src/website/packages/charts/examples/continents/Index.js#L124

Let me know if I missed something. I verified all the other examples and they're working as intended in regards to zooming/panning.

### CODE REVIEW

File: src/components/EventHandler.js [hunk 1]
```diff
@@ -136,7 +132,7 @@ export default class EventHandler extends React.Component {
         document.addEventListener("mouseup", this.handleMouseUp);
 
         this.setState({
-            isPanning: true,
+            isPanning: this.props.enablePanZoom && true,
```
Comment (sartaj10) [inline src/components/EventHandler.js]: `isPanning` can be equal to just `this.props.enablePanZoom`

**Comment by sartaj10** [comment]
@pjm17971 What do you think about this?

File: src/components/EventHandler.js (hunk 1 above)
Comment (guillaume) [reply src/components/EventHandler.js]: Woops, that's right! :-)

Comment (guillaume) [reply src/components/EventHandler.js]: fixed

**Comment by guillaume** [comment]
Any reason not to merge this?

The bug is easy to reproduce. Click on any area chart and then try to unselect it by clicking on the white background. You can't.
image


http://software.es.net/react-timeseries-charts/#/example/continents

**Pull request merged**